OSC/ondemand
# support actually saving scripts

Now in the `scripts#save` it actually, correctly, saves the scripts' form and attributes fields, preserving a number fields min and maxes.  In draft for the moment while I write tests and maybe refactor a little.

To test
* create a script
* add all the fields you can
* edit the min and max of all the number fields (hours and nodes)
* now save
* view the resulting yml file in `.ondeamnd/#{id}.yml` and see that you've saved it correctly
* OR go back to edit it and see that it's preserved the settings that you've saved.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1135148780858012/1204479897192827) by [Unito](https://www.unito.io)
